### DISCUSSION TITLE
Skip to package search

### DISCUSSION THREAD

**Discussion by will-stone** [npm/feedback#543] [Website] [answered]
Hi 👋

Are you currently A/B testing the "skip to package search" functionality when pressing the <kbd>tab</kbd> key of the npm website? I'm finding it either does one of two things:

- Shows a dialog message at the top that says "skip to package search or skip to sign in". Hitting enter then jumps the scroll to the search box but still doesn't focus it, and an extra <kbd>tab</kbd> press is required to get focus on the search box.
   <img width="315" alt="image" src="https://user-images.githubusercontent.com/654103/131518004-f799f2b7-dcc2-4ffd-bcf2-0120f6568465.png">
- Or, pressing <kbd>tab</kbd> immediately focuses the search box.

The latter is preferable as it's only one key press, but to be honest it would just be nice if it was just consistent from day-to-day.

Thanks,

Will

Firefox 91.0.2
macOS 11.5.2

**Comment by MylesBorins** [comment, marked as answer]
hey @will-stone I'm going to let the team working on this know about your feedback

**Reply by will-stone** [reply]
Thank you 😊
